### PR TITLE
feat(observability-lib): improve alerts rule

### PR DESCRIPTION
**Pull request by Atrax1**
- Add function to delete notification template
- Add alert rules filters optional param from cmd
- Simplify provided notification template
- Add alerts for headtracker + services health
image

### CODE REVIEW

**Review by jmank88 — DISMISSED** [review]

File: observability-lib/README.md [hunk 1]
```diff
@@ -1,8 +1,118 @@
...
+    return nil, err
+}
+json, err := db.GenerateJSON()
+if err != nil {
+    return nil, err
+}
+fmt.Println(string(json))
+```
+
+More advanced examples can be found in the `dashboards` folder.
```
Comment (jmank88) [inline observability-lib/README.md:83]: Could these references be made in to relative links?

File: observability-lib/README.md [hunk 2]
```diff
@@ -1,8 +1,118 @@
...
+db, err := builder.Build()
+if err != nil {
+    return nil, err
+}
+json, err := db.GenerateJSON()
+if err != nil {
+    return nil, err
+}
+fmt.Println(string(json))
+```
```
Comment (jmank88) [inline observability-lib/README.md:81]: maybe consider making this an expandable section?
(and a real complete file w/ `func main`?)
<details><summary>main.go</summary>

```go
package main

import "github.com/smartcontractkit/chainlink-common/observability-lib/grafana"

func main() {
	builder := grafana.NewBuilder(&grafana.BuilderOptions{
	    Name:       "Dashboard Name",
	    Tags:       []string{"tags1", "tags2"},
	    Refresh:    "30s",
	    TimeFrom:   "now-30m",
	    TimeTo:     "now",
	})
	
	builder.AddVars(grafana.NewQueryVariable(&grafana.QueryVariableOptions{
	    VariableOption: &grafana.VariableOption{
	        Label: "Environment",
	        Name:  "env",
	    },
	    Datasource: "Prometheus",
	    Query:      `label_values(up, env)`,
	}))
	
	builder.AddRow("Summary")
	
	builder.AddPanel(grafana.NewStatPanel(&grafana.StatPanelOptions{
	    PanelOptions: &grafana.PanelOptions{
	        Datasource:  "Prometheus",
	        Title:       "Uptime",
	        Description: "instance uptime",
	        Span:        12,
	        Height:      4,
	        Decimals:    2,
	        Unit:        "s",
	        Query: []grafana.Query{
	            {
	                Expr:   `uptime_seconds`,
	                Legend: `{{ pod }}`,
	            },
	        },
	    },
	    ColorMode:   common.BigValueColorModeNone,
	    TextMode:    common.BigValueTextModeValueAndName,
	    Orientation: common.VizOrientationHorizontal,
	}))
	
	db, err := builder.Build()
	if err != nil {
	    return nil, err
	}
	json, err := db.GenerateJSON()
	if err != nil {
	    return nil, err
	}
	fmt.Println(string(json))
}
```
</details>

Comment (Atrax1) [reply observability-lib/README.md:81]: yes very good idea 👍

File: observability-lib/README.md (hunk 1 above)
Comment (Atrax1) [reply observability-lib/README.md]: Yes indeed, I just added them

**Review by jmank88 — DISMISSED** [review]

**Review by msuchacz-cll — APPROVED** [review]

File: observability-lib/grafana/variables.go [hunk 3]
```diff
@@ -12,30 +12,44 @@ type VariableOption struct {
...
+	Query        string
+	Multi        bool
+	Regex        string
+	CurrentText  string
+	CurrentValue string
+	IncludeAll   bool
 }
 
 func NewQueryVariable(options *QueryVariableOptions) *dashboard.QueryVariableBuilder {
+	if options.CurrentText == "" {
```
Comment (msuchacz-cll) [inline observability-lib/grafana/variables.go:25]: I think I suggested this before when it came to legends, but can't you do default values in the struct itself?

**Review by patrickhuie19 — APPROVED** [review]

**Pull request merged**